docs/conf/portland/2023/news/announcing-speakers.rst

```diff
@@ -0,0 +1,73 @@
+:template: {{year}}/generic.html
+
+.. post:: February 22, 2023
+   :tags: {{shortcode}}-{{year}}, speakers, tickets, visiting, sponsors
+
+Announcing Portland conference speakers
+=======================================
+
+Our Portland conference is happening in just under 3 months, and we're excited to see everyone again in {{city}}.
+Now is a great time to start booking your tickets and travel,
+and we hope to see you soon.
+
+The biggest announcement today is our speakers and talks for the conference. 
+Thank you so much to everyone who took the time to send us a proposal,
+we literally wouldn't have an event without you!
+
+Full speaker line-up
+--------------------
+
+Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
```

Comment:
Not crucial, but I stumble over this sentence when reading it... maybe s/look to/try to ?